src/List.ts

```diff
@@ -1,8 +1,10 @@
-import { generateID } from "./util/id";
+import { generateId } from "./util/id";
+
+type ScalarValue = string | number | boolean;
```

Comment:
Is this type duplicated between `List.ts` and `Variable.ts`?

Reply:
Yep - as of 7db56d4, the two modules are now combined, since we export multiple things from files like `Target` or `Block` already.